### DISCUSSION TITLE
bad crash application scroll item property - azure

### DISCUSSION THREAD

**Discussion by Paolo-lab** [iTwin/iTwinUI#1455] [Q&A] [answered]
Hello good evening, my end customer noticed a problem with my application located on Azure, I state that I don't have this problem locally, but it only happens on the application located in azure.

when scrolling through the properties of an element the program often crashes, unfortunately it's not code written by me but it's an internal problem I don't know if it's due to azure, maybe undersized or what else I'm attaching two images of the error ..
The customer asks me for explanations, but frankly I don't know what to answer

Thank you, really thank you

<img width="677" alt="azure_error1" src="https://github.com/iTwin/iTwinUI/assets/57446652/e0937bfe-c7a0-4523-a60e-b2f8ea7cad27">

<img width="522" alt="azure_error2" src="https://github.com/iTwin/iTwinUI/assets/57446652/f5a471bd-c9ad-40f2-8424-dffc113868cf">

<img width="195" alt="Package" src="https://github.com/iTwin/iTwinUI/assets/57446652/4429be1f-f75c-4fe4-9903-93d2657b2566">

**Comment by mayank99** [comment, marked as answer]
this is not related to iTwinUI at all. you can see in the stacktrace the filenames where it's happening, none of those come from iTwinUI
<img width="758" alt="" src="https://github.com/iTwin/iTwinUI/assets/9084735/a13e5c6e-8335-4785-b061-496e8c3fa329">

**Reply by r100-stack** [reply]
@Paolo-lab Since you added a thumbs up to the above answer, I marked it as the answer. But feel free to continue the conversation if your answer is still not resolved.
